src/components/LiquidityMining/Create/Steps/PreviewAndCreate/SimulateStaking/index.tsx

```diff
@@ -0,0 +1,229 @@
+import { ReactComponent as CryptoIcon } from '../../../../../../assets/svg/crypto-icon.svg'
+import { ReactComponent as RefreshIcon } from '../../../../../../assets/svg/refresh-icon.svg'
+import NumericalInput from '../../../../../Input/NumericalInput'
+import { TYPE } from '../../../../../../theme'
+
+const SwitchContainer = styled(Flex)`
+  font-size: 10px;
+  display: flex;
+  font-weight: 600;
+  color: ${props => props.theme.text5};
+  line-height: 11px;
+  letter-spacing: 0.15em;
+  text-transform: uppercase;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  gap: 4px;
+`
+const SimulatedValue = styled.div`
+  font-family: 'Fira Mono';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 22px;
+  /* identical to box height */
+
+  text-align: right;
+  letter-spacing: 0.02em;
+  text-transform: uppercase;
+
+  color: ${props => props.theme.text2};
+`
+const AmountFlex = styled(Flex)`
+  width: max-content;
+  background: #000000;
+  backdrop-filter: blur(20px);
+  border-radius: 4px;
+  padding: 8px 12px;
+  align-self: center;
+`
+
+const SimulateOption = styled.div<{ isActive: boolean }>`
+  font-size: 11px;
+  font-weight: 600;
+  letter-spacing: 0.08em;
+  color: ${props => (props.isActive ? props.theme.text2 : props.theme.lightPurple2)};
+  ${props => props.isActive && 'text-decoration: underline; text-underline-offset: 7px;'};
+`
+const StyledNumericalInput = styled(NumericalInput)<{ value: string }>`
+  color: ${props => props.theme.text2};
+  width: ${({ value }) => (value && value.length > 24 ? '250' : value ? 17 + value?.length * 8 : 20)}px;
+  max-height: 38px;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 16px;
+  text-transform: uppercase;
+  ::placeholder {
+    color: ${props => props.theme.text2};
+  }
+`
+
+const DOLLAR_AMOUNT_MAX_SIMULATION = 10000000
+
+interface RewardSummaryProps {
+  tokenOrPair: Token | Pair | null
+  stakingCap: TokenAmount | null
+  nativeCurrencyUSDPrice: Price
+  setSimulatedStakedAmount: (value: string) => void
+  setSimulatedPrice: (value: any) => void
+  simulatedPrice: any
+  loading: boolean
+}
+enum SimulateOptions {
+  AMOUNT = 'Amount',
+  PRICE = 'Price',
+}
+
+export default function SimulateStaking({
+  tokenOrPair,
+  stakingCap,
+  setSimulatedStakedAmount,
+  setSimulatedPrice,
+  simulatedPrice,
+  nativeCurrencyUSDPrice,
+  loading,
+}: RewardSummaryProps) {
+  const { loading: loadingNativeTokenPrice, derivedNativeCurrency: nativeTokenPrice } = useTokenOrPairNativeCurrency(
+    tokenOrPair ? tokenOrPair : undefined
+  )
+
+  const handleLocalStakingCapChange = useCallback(
+    rawValue => {
+      setSimulatedPrice(rawValue)
+    },
+    [setSimulatedPrice]
+  )
+  useEffect(() => {
+    if (nativeTokenPrice && nativeCurrencyUSDPrice)
+      setSimulatedPrice(nativeTokenPrice.multiply(nativeCurrencyUSDPrice).toFixed(2))
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
If we are defining this, something is wrong here. This line shouldn't exist and must be deleted.